constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -94,38 +97,45 @@ type RegoRewriterTestcase struct {
 	name       string            // testcase name
 	baseSrcs   map[string]string // entrypoint files
 	libSrcs    map[string]string // lib files
-	wantError  bool              // true if RegoRewriter should reject input
+	wantError  error             // error returned when RegoRewriter should reject input
 	wantResult map[string]string // expected output files from RegoRewriter
 }
 
-func (tc *RegoRewriterTestcase) Run(t *testing.T) {
+func MockRegoWriter() (*RegoRewriter, error) {
```

Comment:
It's better for tests to be explicit about the prefixer, libs, and externs they rely on. As-is the test cases give no hint that these values are special. It would be better to repeat the content of this method in each test so that (1) it is clear that these values have some special meaning in the tests and (2) it is easier to make the tests more flexible, e.g. if we want to test different values of `libs` or `externs`.

Reply:
Specified `libs` and `externs` are tested below in TestRegoRewriterRewriteDataRef, tested `prefix` in TestRegoRewriteLib.  Kept `MockRegoWriter` for test cases with fixed regorewriter initializer.